### ISSUE TITLE
TextAlign property doesn't work for ToolStripMenuItem

### ISSUE THREAD

**Issue by John-Qiao** [dotnet/winforms#4985]
* .NET Core Version: 

   .NET SDK 6.0.100-preview.6.21276.13

* Have you experienced this same bug with .NET Framework?: 

   Yes

**More Info:**

1. This is not the regression issue, it occurs in .NET Framework project too.
2. This issue occurs in the design time too.
3. This issue occurs in MenuStrip/ContextMenuStrip/StatusStrip/ToolStrip controls.

**Problem description:**

TextAlign property doesn't work for **File** item.
![Actual](https://user-images.githubusercontent.com/45864985/119808611-dfcef480-bf16-11eb-84a2-be4db56a7676.png)

**Expected behavior:**

As set the TextAlign = MiddleRight for the **File** item in form designer, so the text of **File** item should shows on the right side.
![PropertySetting](https://user-images.githubusercontent.com/45864985/119808486-be6e0880-bf16-11eb-9955-156605323a55.png)

**Minimal repro:**

1. Open the attached [TestApp.zip](https://github.com/dotnet/winforms/files/6553044/TestApp.zip) project in VS and run it.
2. Click on **toolStripMenuItem1** to expand its drop down item **File**.
3. Observe the text align of **File** item.

**Comment by dreddy-work** [comment]
@John-Qiao , Is this issue in response to a feedback? We would need a business case to implement functionality. In the best case we may hide design time property.  Did you check behavior in MFC (C++) app with 'MainMenu'?

**Comment by John-Qiao** [comment]
@dreddy-work  yes, it's a feedback issue which post in here: https://developercommunity2.visualstudio.com/t/ToolStripMenuItem-TextAlign-doesnt-work/856122.

And I didn't find the TextAlign property for Menu items in MFC app, please check below screenshot:
![MFC-Main](https://user-images.githubusercontent.com/45864985/119949118-229ed400-bfcc-11eb-8feb-260f225257bc.png)

**Comment by RussKie** [comment]
There is a solution available on SO https://stackoverflow.com/a/4397976/2338036, which also notes that "_this is not standard Windows GUI menu item layout which users expect_".
 Our goal is to allow building Windows native apps, and this request is asking to allow building apps that don't look native on Windows. 

I propose we hide these API from the strip controls. Any users that desire to build apps with non-traditional Windows look and feel can provide custom renderers and achieve the desired.
